cimetrics/env.py

```diff
@@ -4,14 +4,20 @@
 from typing import Optional
 import os
 import yaml
-from git import Repo
+from git import Repo, exc
 
 
 def get_env():
+    try:
+        repo = Repo(os.getcwd(), search_parent_directories=True)
+    except exc.InvalidGitRepositoryError:
+        print("Environment is not a valid git repository.")
```

Comment:
I think this should print the pwd instead of "Environment", "/blah/foo is not a valid git repository".